<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Thembani Moithobogi</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>480</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>CFT Zoning Tool</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/cft.ico</normaloff>icon/cft.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>547</width>
     <height>580</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>547</width>
     <height>580</height>
    </size>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>550</width>
      <height>450</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>550</width>
      <height>450</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>550</width>
      <height>450</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>0</y>
       <width>525</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>525</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>525</width>
       <height>50</height>
      </size>
     </property>
     <property name="title">
      <string>Output Directory</string>
     </property>
     <widget class="QPushButton" name="outputButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>93</width>
        <height>25</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="outdirlabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>381</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>381</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>381</width>
        <height>25</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="runButton">
     <property name="geometry">
      <rect>
       <x>408</x>
       <y>420</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>50</y>
       <width>525</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>525</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>525</width>
       <height>50</height>
      </size>
     </property>
     <property name="title">
      <string>Input Geospatial Layer</string>
     </property>
     <widget class="QPushButton" name="InputLayerButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>93</width>
        <height>25</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="inputlayerlabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>381</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>381</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>381</width>
        <height>25</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>260</y>
       <width>240</width>
       <height>150</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>240</width>
       <height>150</height>
      </size>
     </property>
     <property name="title">
      <string>Climate Element properties</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Format:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>20</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <widget class="QRadioButton" name="NetCDFRadio">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>74</x>
           <y>0</y>
           <width>54</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>NC</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="CSVRadio">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>54</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>CSV</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_14">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Missing:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="missingvalueslineEdit">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_12">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Function.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QFrame" name="frame_3">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>20</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <widget class="QRadioButton" name="cumRadio">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>53</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Sum</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="avgRadio">
         <property name="geometry">
          <rect>
           <x>73</x>
           <y>0</y>
           <width>55</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Mean</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_10">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Period:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="periodComboBox">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>258</x>
       <y>260</y>
       <width>270</width>
       <height>150</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>270</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>270</width>
       <height>150</height>
      </size>
     </property>
     <property name="title">
      <string>Climate Element data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QListWidget" name="predictandlistWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>70</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="browsepredictandButton">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="clearpredictandButton">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="predictandparamcombobox">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>258</x>
       <y>100</y>
       <width>270</width>
       <height>160</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>270</width>
       <height>160</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>270</width>
       <height>160</height>
      </size>
     </property>
     <property name="title">
      <string>Climatology</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>End Year:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="ZonesLineEdit">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="endyearLineEdit">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="startyearLineEdit">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>No. of Zones:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Start Year:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Climatological reference</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>240</width>
       <height>160</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>160</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>240</width>
       <height>160</height>
      </size>
     </property>
     <property name="title">
      <string>Zoning Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Interpolation:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButtonIDW">
        <property name="text">
         <string>IDW</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="radioButtonLinear">
        <property name="text">
         <string>Linear</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Max. PEV</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="PEV">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Gridsize:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="gridsizelineEdit">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Rotation</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="rotationDcombobox">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="statusbarlabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>385</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>385</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>385</width>
       <height>25</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>43</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>547</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>547</width>
     <height>22</height>
    </size>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
